metagraph/src/graph/representation/succinct/dbg_succinct.hpp

```diff
@@ -183,6 +185,8 @@ class DBGSuccinct : public DeBruijnGraph {
 
     const mtg::kmer::KmerBloomFilter<>* get_bloom_filter() const { return bloom_filter_.get(); }
 
+    std::shared_ptr<DBGSuccinctCached> get_cached_view(size_t cache_size = 1024) const;
```

Comment:
Why so small default cache?

Add a comment how many bytes are taken by one element in the cache.

Reply:
When I was testing this, it wasn't getting more efficient after 1024 elements, but I think that's just because I was testing with a small, non-redundant short read set